src/main/java/com/marginallyclever/makelangelo/Makelangelo.java

```diff
@@ -309,30 +309,22 @@ private JMenu createToolsMenu() {
 
 		try {
 			PiCaptureAction pc = new PiCaptureAction();
-			
-			if(pc != null) {
-				JButton bCapture = new JButton(Translator.get("MenuCaptureImage"));
-				bCapture.addActionListener((e)->{
-					pc.run(mainFrame,myPaper);
-				});
-		        menu.add(bCapture);
-				menu.addSeparator();
-		    } 
+
```

Comment:
PiCapture action can be null here and this change will fail.

Reply:
It is a constructor, if it fails, it will throw an exception, if it succeed the object `PiCaptureAction` will be instantiated.